docs/src/pages/[platform]/index.page.tsx

```diff
@@ -293,7 +293,7 @@ const HomePage = ({ colorMode }) => {
                   WCAG
                 </Link>{' '}
                 and{' '}
-                <Link isExternal href="https://www.w3.org/TR/wai-aria-1.2/">
+                <Link isExternal href="https://www.w3.org/WAI/ARIA/apg/">
```

Comment:
I know we're updating the home page soon, but updating this link to the new one in the meantime.